other/median_of_two_arrays.py

```diff
@@ -0,0 +1,31 @@
+# This code finds the median of two arrays, even if they are not sorted initially
+def findMedianArrays(nums1, nums2):
```

Comment:
```suggestion
def find_median_arrays(nums1, nums2):
```
Python naming as discussed in CONTRIBUTING.md.  Please add type hints on this line and doctests.